CHANGELOG.md

```diff
@@ -0,0 +1,350 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [v1.0.0](https://github.com/voxpupuli/container-puppetserver/tree/v1.0.0) (2023-11-13)
+
+[Full Changelog](https://github.com/voxpupuli/container-puppetserver/compare/7.13.0...v1.0.0)
+
+**Closed issues:**
+
+- Puppetserver container manifest unknown [\#17](https://github.com/voxpupuli/container-puppetserver/issues/17)
+- Puppetserver 8 [\#16](https://github.com/voxpupuli/container-puppetserver/issues/16)
+
+**Merged pull requests:**
+
+- Update build\_arch to include linux/arm64 [\#21](https://github.com/voxpupuli/container-puppetserver/pull/21) ([rwaffen](https://github.com/rwaffen))
+- Add matrix setup to build several versions in parallel specified in a json file [\#19](https://github.com/voxpupuli/container-puppetserver/pull/19) ([rwaffen](https://github.com/rwaffen))
+
+## [7.13.0](https://github.com/voxpupuli/container-puppetserver/tree/7.13.0) (2023-10-19)
```

Comment:
Major version is going backwards?

Reply:
yes i want to break with the current tags. the old git tags are bound to the puppet version, but i thought it might be good to have our own git and container tags, so we can release more often then puppet versions are being released. so we don't have to move refs and confuse people if we rebuild the container, but the puppet version stays the same.